.gitignore

```diff
@@ -19,6 +19,7 @@ build
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+yarn.lock
```

Comment:
No, we need `yarn.lock` in the repository to manage dependencies. Please remove this line.

```suggestion
```

Instead, do not make changes to `yarn.lock`, and do not commit changes to it. You can add it to your user's `.gitignore` instead of the project's, if it really bothers you.